tests/acceptance/echo_feature_bash.sh

```diff
@@ -47,7 +47,11 @@ printTestName "ECHO"
 runWithoutQuotes "echo Hello" "echo Hello" "Simple String"
 runWithoutQuotes "echo Hello      Hello" "echo Hello      Hello" "String with spaces to be trimmed"
 runWithoutQuotes "echo -n-n Hello" "echo -n-n Hello" "String with invalid n flag"
+runWithoutQuotes "echo $PWD" "echo \$PWD" "With env var"
+runWithQuotes 'Hello      Hello $PWD' "echo 'Hello      Hello \$PWD'" "String with single quotes"
 runWithQuotes "Hello      Hello" "echo \"Hello      Hello\"" "Quoted string, shouldn't trim"
+runWithQuotes "Hello      Hello $PWD " "echo \"Hello      Hello \$PWD \"" "Quoted string, shouldn't trim and should get var name"
+runWithQuotes "Hello      Hello $PWDs " "echo \"Hello      Hello \$PWDs \"" "Quoted string, shouldn't trim, and variable is invalid"
 runWithQuotes "                Hello      Hello" "echo \"                Hello      Hello\"" "Quoted string, shouldn't trim"
 
```

Comment:
The single $ var token ;)) is being skipped when printing 
to be added to test:
`echo $`
`echo $USER$`